public/app/themes/justice/src/sass/admin-bar-logo.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Override the admin-bar logo
+ */
+div#wpadminbar #wp-admin-bar-wp-logo {
```

Comment:
Minor, if we can trust WP to have unique ids, then we can delete `div#wpadminbar ` here.

Reply:
oh, nice :)